errext/print.go

```diff
@@ -0,0 +1,30 @@
+package errext
+
+import (
+	"errors"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Fprint formats the given error and writes it to l.
+// In case of Exception, it uses the stack trace as the error message.
+// In case of HasHint, it also adds the hint as a field.
+func Fprint(l logrus.FieldLogger, err error) {
```

Comment:
I could add more fields as a variadic argument (might be useful for future uses), so in this case I could add a `msg` telling the user that despite of the exception, it's falling back to the default summary.